frontend/assets/javascripts/src/modules/analytics/ga.es6

```diff
@@ -0,0 +1,76 @@
+import * as user  from 'src/utils/user'
+import * as cookie from 'src/utils/cookie'
+
+const dimensions = {
+    signedIn: 'dimension1',
+    signedOut: 'dimension2',
+    ophanPageViewId: 'dimension3',
+    ophanBrowserId: 'dimension4',
+    platform: 'dimension5',
+    identityId: 'dimension6',
+    isLoggedOn: 'dimension7'
+};
+
+function create(){
+    const tracker = 'membershipPropertyTracker';
+    ga('create', guardian.googleAnalytics.trackingId, {
```

Comment:
It will be available, but it's potentially an unpleasant amount of global nonsense.